pkg/network/ebpf/c/protocols/http/http.h

```diff
@@ -215,6 +215,9 @@ int socket__http_filter(struct __sk_buff* skb) {
 
     read_into_buffer_skb((char *)event.http.request_fragment, skb, skb_info.data_off);
     http_process(&event, &skb_info, NO_TAGS);
+    if (is_direct_flush_supported()) {
+        http_batch_flush(skb);
+    }
```

Comment:
1. Let's do that for http2 and kafka socket filters as well
2. Should we disable `tracepoint__net__netif_receive_skb` hook on kernels 5.4+?

Reply:
1. Looks like calling `http2_batch_flush()` is blowing up the eBPF stack limit, so I'd rather tackle this in a separate PR

2. In _theory_ I think we can do that, but I'd also tackle this in a separate PR if that sounds OK to you.
Leaving `tracepoint__net__netif_receive_skb` as it is now will likely amount to an extra safe-guard, so if there are certain code paths that are not necessarily flushing data to userspace (say, for example, a GnuTLS uprobe), they would still "piggyback" on the tracepoint execution.